### ISSUE TITLE
NEST SQL Translate no aggregations returned

### ISSUE THREAD

**Issue by ipxnsv** [elastic/elasticsearch-net#5201]
**NEST/Elasticsearch.Net version**: 7.01

**Elasticsearch version**:

SQL:
"select UDFvarchar1, count(1) from interactions group by UDFvarchar1 order by UDFvarchar1 "
Kibana works with this SQL and translate returns correctly

Calling Translate:
var q2 = new TranslateSqlRequest()
{
Query = "select UDFvarchar1, count(1) from interactions group by UDFvarchar1 order by UDFvarchar1 "

};
var sqlResponse2 = _client.Sql.Translate(q2);
var reqs = sqlResponse2.Result

Reqs.Aggregations is null

**Comment by russcam** [comment]
Here's a replication

```c#
private static void Main()
{
    var pool = new SingleNodeConnectionPool(new Uri($"http://localhost:9200"));
    var settings = new ConnectionSettings(pool, new InMemoryConnection(Encoding.UTF8.GetBytes(@"{
          ""size"" : 0,
          ""_source"" : false,
          ""stored_fields"" : ""_none_"",
          ""aggregations"" : {
            ""groupby"" : {
                ""composite"" : {
                    ""size"" : 1000,
                    ""sources"" : [
                        {
                            ""ccf51bfa"" : {
                                ""terms"" : {
                                    ""field"" : ""UDFvarchar1.keyword"",
                                    ""missing_bucket"" : true,
                                    ""order"" : ""asc""
                                }
                            }
                        }
                    ]
                }
            }
        }
    }")));
        
    var client = new ElasticClient(settings);
    
    var translateResponse = client.Sql.Translate(t => t
        .Query("select UDFvarchar1, count(1) from interactions group by UDFvarchar1 order by UDFvarchar1")
    );

    var aggs = translateResponse.Result.Aggregations;
    Console.WriteLine($"aggs is null: {aggs is null}");
}
```

**Comment by stevejgordon** [comment]
Sorry for the delay on my side. Thanks for reporting this @ipxnsv and thanks @russcam for the really helpful repro code. I've been playing with this and have a potential fix in place. I want to check it out a bit more on my side but we can aim to get this into the next release.

**Comment by stevejgordon** [comment]
Fixed in #5228 and will be available in 7.11.0
